doc/nrf/releases_and_maturity/releases/release-notes-2.7.99-cs1.rst

```diff
@@ -0,0 +1,86 @@
+.. _ncs_release_notes_2799_cs1:
+
+|NCS| v2.7.99-cs1 Release Notes
+###############################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| v2.7.99-cs1 is a **customer sampling** release, tailored exclusively for participants in the nRF54H20 SoC customer sampling program.
+Do not use this release of |NCS| if you are not participating in the program.
+
+The release is not part of the regular release cycle and is specifically designed to support early adopters working with the nRF54H20 SoC.
+It is intended to provide early access to the latest developments for nRF54H20 SoC, enabling participants to experiment with new features and provide feedback.
+
+The scope of this release is intentionally narrow, concentrating solely on the nRF54H20 DK.
+You can use the latest stable release of |NCS| to work with other boards.
+
+All functionality related to nRF54H20 SoC is considered experimental.
+
+Highlights
+**********
+
+This release introduces following features:
```

Comment:
Could we have this modified as
```suggestion
Added the following features as supported:
```
similar to the entries in the older release notes.